index.d.ts

```diff
@@ -0,0 +1,18 @@
+import { cleanup, act, RenderOptions, RenderResult } from 'react-testing-library'
+
+export function renderHook<P extends any, R extends any>(
```

Comment:
what is the difference between `renderHook<P extends any, R extends any>` and `renderHook<P, R>`?

Reply:
`rednerHook<P, R>` is the same as `renderHook<P extends Object, R extends Object>`.
Maybe it's more simple to use `<P, R>` instead of `<P extends any, R extends any>` since `P` and `R` will be inferred by giving `callback` & `initialProps`.